### ISSUE TITLE
Lab9: Autoscaling is not working for me

### ISSUE THREAD

**Issue by dxz149** [openfaas/workshop#107]
Hi all,
I've been trying to run lab9.
But my autoscaling does not work and it always gives me 1 replica.
Do you know where should I check the value of com.openfaas.scale.min and com.openfaas.scale.max?
And how should I change it? I think the default value for me is to have one replica which never terminates.

**Comment by alexellis** [comment]
Hi,

What is your configuration?

Swarm or Kubernetes?

What versions?

How do you know it's not triggering?

Alex

**Comment by dxz149** [comment]
I use Swarm,
I use Windows with gitbash and faas is version 0.8.3
Then I tried to follow the same instructions for my macbook and autoscaling worked (but after about 5 minutes). 
I checked both http://127.0.0.1:9090/graph and the instruction "faas-cli list" to see the number of replicas and it's always 1 for all functions on windows.
I wonder if there is any way that I can change the scheduler for autoscaling.
Thanks
